### ISSUE TITLE
E402 regression

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#739]
In GitLab by @Dreamsorcerer on May 20, 2020, 14:47

With some import statements like this:
```
import gbulb
import gi
gi.require_version("Gtk", "3.0")  # noqa: E402
from gi.repository import Gio, GLib, Gtk
import tesla_api
```

In 3.7.9, no E402 violations are reported.

In 3.8.1, both of the last 2 lines report violations.

`E402 module level import not at top of file`

I hope that change can be reverted, as requiring every import line to have the noqa tag would make this rule too much hassle to use.

**Comment by asottile** [comment]
In GitLab by @Dreamsorcerer on May 20, 2020, 14:48

changed the description

**Comment by asottile** [comment]
In GitLab by @ericvw on May 20, 2020, 14:57

This is a duplicate of #638.

Code bases will need to adapt to this change.  It's unfortunate that !638, which fixes issues with checkers hiding `# noqa` lines from `flake8`, changes unintended relied upon behavior.

**Comment by asottile** [comment]
In GitLab by @Dreamsorcerer on May 20, 2020, 15:18

That's annoying, is there any proposal to get this specific use case back in?

That snippet is the middle of my imports, there's another block of imports after that, and every line would need the exception. I'll probably just disable E402 globally, if this can't be fixed, as it's too messy otherwise.

**Comment by asottile** [comment]
In GitLab by @asottile on May 20, 2020, 16:54

closed
